service/service_test.go

```diff
@@ -166,7 +171,7 @@ func TestRun(t *testing.T) {
 
 			Convey("The checkers are registered and the healthcheck and http server started", func() {
 				So(len(hcMock.AddCheckCalls()), ShouldEqual, 1)
-				So(len(initMock.DoGetHTTPServerCalls()), ShouldEqual, 1)
+				So(len(initMock.DoGetHTTPServerCalls()), ShouldEqual, 2)
 				So(initMock.DoGetHTTPServerCalls()[0].BindAddr, ShouldEqual, ":29100")
```

Comment:
Now that there are 2 calls instead of one, you could add a check for the second call like it's being done here.